js_modules/dagster-ui/packages/ui-core/src/runs/RunTable.tsx

```diff
@@ -278,24 +278,38 @@ const RunRow: React.FC<{
     }
   };
 
+  const [unpinnedTags, setUnpinnedTags] = useStateWithStorage('unpinned-tags', (value) => {
```

Comment:
we're storing the unpinned tags?  does this mean that all new tags show up by default?

Reply:
Yes exactly